docs/supported-platforms.md

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
I'd delete this file because (1) we don't need the Tizen column, and (2) the macOS column is about macOS rather than Mac Catalyst. So that leaves everything supported on our platforms.